packages/gallery/src/components/gallery/leanGallery/leanGallery.scss

```diff
@@ -1,10 +1,11 @@
 :global {
   div.lean-gallery-gallery {
     display: grid;
+    position: static;
```

Comment:
why?

Reply:
Here it is just a small discussion I wanted to talk to you.

This change is not mandatory, 
but when using "display: grid" mechanism we dont need to use the "position" property at all,
and because we still use the classes of pro-gallery, I wanted to highlight that we can override the position property to its default value.